platforms/pf_test/platform.cpp

```diff
@@ -32,6 +34,9 @@
 #define ENABLE_DEBUG        (0)
 #include "debug.h"
 
+/// Maximum number of arguments to shell command callbacks
+#define MAX_COMMAND_ARGS 16
```

Comment:
16 is not too much ? 8 should be enough ?

Reply:
Yes, 8 is enough.